public/components/ContentPanel/ContentPanel.js

```diff
@@ -28,9 +21,9 @@ const ContentPanel = ({
   <EuiPanel style={{ paddingLeft: '0px', paddingRight: '0px', ...panelStyles }}>
     <EuiFlexGroup style={{ padding: '0px 10px' }} justifyContent="spaceBetween" alignItems="center">
       <EuiFlexItem>
-        <EuiTitle size={titleSize}>
-          <h3>{title}</h3>
```

Comment:
Can you point out which screenshot this is? I guess because its in a panel, not sure where its being used and if moving from H3->H2 makes sense.

If this is a "tile header", the move makes sense

Reply:
it's a panel header, and it's being used in many places, see below screenshot
![image](https://github.com/user-attachments/assets/2d940c9e-1d69-4c69-882d-dae041cf8d28)
![image](https://github.com/user-attachments/assets/68d43969-1d54-4487-866e-c65fc214ef16)